gluster/block/cmd/glusterblock-provisioner/glusterblock-provisioner.go

```diff
@@ -319,6 +301,32 @@ func (p *glusterBlockProvisioner) Provision(options controller.VolumeOptions) (*
 	return pv, nil
 }
 
+func (p *glusterBlockProvisioner) storeFieldsFromResponse(blockVolName string, cfg *provisionerConfig, blockVol *glusterBlockVolume, iscsiVol *iscsiSpec) error {
+	//Store fields from response to iscsiSpec struct
+	if cfg.opMode == heketiOpmode && blockVol.heketiBlockVolRes != nil {
```

Comment:
validate cfg != nil and glusterBlockVolume

Reply:
Done.